app/Repositories/Eloquents/BookEloquentRepository.php

```diff
@@ -251,13 +251,13 @@ public function getRelatedBooks($bookIds, $with = [])
             ->get();
     }
 
-    public function search($attribute, $data, $with = [])
+    public function search($attribute, $data, $with = [], $page)
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PEAR.Functions.ValidDefaultValue.NotAtEnd</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>**Line**: 254</br>Arguments with default values must be at the end of the argument list